_includes/snippets/13/statu_output.snip

```diff
@@ -1,2 +1,8 @@
+{: .bash}
```

Comment:
This `{:bash}` tag belongs to the preceding block. Please remove from this snippet.